python/tchannel/tornado/tchannel.py

```diff
@@ -226,23 +226,17 @@ def __init__(self, hostport, service, tchannel):
     @tornado.gen.coroutine
     def send(self, arg_1, arg_2, arg_3):
         peer_connection = yield self.tchannel().get_peer(self.hostport)
-        self.message_id = message_id = peer_connection.next_message_id()
+        self.message_id = peer_connection.next_message_id()
```

Comment:
Actually, I don't think `self.message_id` is used anywhere. I would remove the attribute completely and just track the message ID in a local variable.